index.js

```diff
@@ -1,6 +1,41 @@
 /* jshint node: true */
 'use strict';
 
+var path = require('path');
+var mergeTrees = require('broccoli-merge-trees');
+
 module.exports = {
-  name: 'ember-redux'
+  name: 'ember-redux',
+
+  included: function(app) {
+    // Import UMD redux and redux-thunk
+    // transform to named AMD that we can import
+    app.import('vendor/redux.js', {
+      using: [{ transformation: 'amd', as: 'redux' }]
+    });
+    app.import('vendor/redux-thunk.js', {
+      using: [{ transformation: 'amd', as: 'redux-thunk' }]
+    });
+    // If developing the addon itself, add redux-saga
+    if (this.parent.pkg.name === 'ember-redux') {
+      app.import('vendor/redux-saga.js', {
+        using: [{ transformation: 'amd', as: 'redux-saga' }]
+      });
+    }
+  },
+
+  treeForVendor: function() {
+    // Grab the UMD distributions of redux and redux-thunk
+    var trees = [
+      path.join(path.dirname(require.resolve('redux')), '..', 'dist'),
+      path.join(path.dirname(require.resolve('redux-thunk')), '..', 'dist'),
+    ];
+    // If developing the addon itself, add redux-saga
+    if (this.parent.pkg.name === 'ember-redux') {
+      trees.push(
+        path.join(path.dirname(require.resolve('redux-saga')), '..', 'dist')
+      );
+    }
+    return mergeTrees(trees);
```

Comment:
I'm not sure if we also need to merge the trees coming from `_super`. It doesn't seem to be used today, but if we had any bower dependency or files inside of ember-redux/vendor, I think that's required. Optionally, it may be simpler to use https://github.com/kellyselden/ember-cli-node-modules-to-vendor instead.